CMakeLists.txt

```diff
@@ -277,8 +281,22 @@ if (TELEMETRY_MONITOR)
     add_definitions(-DTELEMETRY_MONITOR)
 endif()
 
-if (ENABLE_EXTENDED_POLICY)
-    add_definitions(-DEXTENDED_POLICY)
+if (${EXTENDED_POLICY} STREQUAL "EXTENDED_PROPRIETARY")
+    add_definitions(-DEXTENDED_PROPRIETARY)
+    set(POLICY_PATH ${COMPONENTS_DIR}/policy/policy_premium/)
+    set(POLICY_GLOBAL_INCLUDE_PATH ${COMPONENTS_DIR}/include/policy/policy_premium/)
```

Comment:
@LuxoftAKutsan 
For consistency I would be good to rename folders to 'extended_proprietary'